docs/settings_dict_guide.md

```diff
@@ -113,6 +113,7 @@ The algorithm will run faster and use less resources if this is set to false.
 
 A list specifying how records should be compared for probabalistic matching.  Each element is a dictionary
 
+*Note*: Dictionaries must be defined with double quotes as if they're valid json. Single quoted keys and values will fail validation. 
```

Comment:
Would you mind removing this comment from here? 

I've asked Robin to move it across to [the settings editor](https://www.robinlinacre.com/splink3_settings_editor_temp/), as I believe this is a better location for the note.

Reply:
Yep - I had originally thought about leaving the comment out. 

If I had more time I'd dig into the `validate_jsonschema.py` and see if there was a more elegant solution. I don't think I've ever had an issue converting a python dict to json before so this was a first.